RemoteHealthcare/SimulatorBike.cs

```diff
@@ -25,22 +18,16 @@ public void SetResistance(byte resistance)
         public void SetAirResistance(byte airResistanceCoefficient, byte windspeed, byte draftingFactor)
         {
             double windResistance = (draftingFactor * windspeed * airResistanceCoefficient);
+
             // If windresistance is 0 then total resistance is 0 as well.
             // If windresistance > 0 then determine how much windresistance there is compared to the max possible (65.62).
             // Which is determined from max wind speed * max wind resistance coefficient * max drafting factor = 127 * 1.86 * 1
             // This then gives a value of 0 to 1, which is multiplied by 256 to get a hex value of 0x00 to 0xFF,
             // which is used for the total resistance.
             this.resistance = windResistance <= 0 ? (byte)0 : (byte)(65.62 / windResistance * 256);
         }
-        
 
-        public void startSim()
-        {
-            Run();
-            return;
-        }
-
-        public void Run()
+        public void StartSim()
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)

Reply:
Name convention zelfde functie naam als in consoleGUI.cs